doc/specs/vulkan/chapters/memory.txt

```diff
@@ -552,58 +552,57 @@ There must: be at least one memory type with the
 ename:VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT bit set in its
 pname:propertyFlags.
 
-The memory types are sorted according to a preorder which serves to aid in
-easily selecting an appropriate memory type.
-Given two memory types X and Y, the preorder defines [eq]#X {leq} Y# if:
-
-  * the memory property bits set for X are a strict subset of the memory
-    property bits set for Y.
-    Or,
-  * the memory property bits set for X are the same as the memory property
-    bits set for Y, and X uses a memory heap with greater or equal
-    performance (as determined in an implementation-specific manner).
-
-Memory types are ordered in the list such that X is assigned a lesser
-pname:memoryTypeIndex than Y if [eq]#(X {leq} Y) {land} {lnot} (Y {leq} X)#
-according to the preorder.
-Note that the list of all allowed memory property flag combinations above
-satisfies this preorder, but other orders would as well.
-The goal of this ordering is to enable applications to use a simple search
-loop in selecting the proper memory type, along the lines of:
+The array pname:memoryTypes must: be ordered in such a way that for each of
```

Comment:
nit: I'd probably just start this sentence "pname:memoryTypes must: be..." - "The array" is just extra unnecessary verbage, and is a bit jarring as it's inconsistent with the rest of the spec.

Reply:
Was that before. I did not like that because it make the sentence start with lower-case.

Reply:
Yea it's a bit weird, but it's an identifier so it's fine - we do this all over the place in the spec.